docs/integrations/app-integrations/microsoft-teams.mdx

```diff
@@ -7,53 +7,162 @@ In this section, we present how to connect Microsoft Teams to MindsDB.
+The Microsoft Temas handler allows you to read and send messages to a Microsoft Teams channel or chat. It also allows you to create a chatbot that will respond to messages.
 
 ## Connection
 
-This handler was implemented using the [pymsteams](https://pypi.org/project/pymsteams/), which is the Python Wrapper Library to send requests to Microsoft Teams Webhooks.
+This handler was implemented using [msal](https://github.com/AzureAD/microsoft-authentication-library-for-python) for authentication and [Requests](https://github.com/psf/requests) to submit requests to the Microsoft Graph API.
 
-The Microsoft Temas handler is initialized with a single parameter:
+The Microsoft Teams handler is initialized with the following parameters:
 
-* `webhook_url` is a webhook url for the Microsoft Teams channel to send messages to.
+- `client_id`: The client ID of the registered Microsoft Entra ID application.
+- `client_secret`: The client secret of the registered Microsoft Entra ID application.
+- `tenant_id`: The tenant ID of the registered Microsoft Entra ID application.
 
 <Note>
-Read about creating an Incoming Webhook in Microsoft Teams [here](https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?tabs=dotnet).
+Microsoft Entra ID was previously known as Azure Active Directory (Azure AD).
 </Note>
 
+The parameters given above can be obtained by registering an application in Entra ID by following these steps,
+1. Go to the [Azure Portal](https://portal.azure.com/#home) and sign in with your Microsoft account.
+2. Locate the **Microsoft Entra ID** service and click on it.
+3. Click on **App registrations** and then click on **New registration**.
+4. Enter a name for your application and select the **Accounts in this organizational directory only** option for the **Supported account types** field.
+5. Keep the **Redirect URI** field empty and click on **Register**.
+6. Copy the **Application (client) ID** and paste it as the `client_id` parameter, and copy the **Directory (tenant) ID** and paste it as the `tenant_id` parameter.
+7. Click on **Certificates & secrets** and then click on **New client secret**.
+8. Enter a description for your client secret and select an expiration period.
+9. Click on **Add** and copy the generated client secret and paste it as the `client_secret` parameter.
+10. Click on **Authentication** and then click on **Add a platform**.
+11. Select **Web** and enter the following URLs in the **Redirect URIs** field:
+    - `https://cloud.mindsdb.com/verify-auth`
+    - `http://localhost:47334/verify-auth` (for local development)
+
+<Tip>
+You can find more information about creating app registrations [here](https://docs.microsoft.com/en-us/graph/auth-register-app-v2).
+</Tip>
+
+When the above is statement is executed with the given parameters, the handler will open a browser window and prompt you to sign in with your Microsoft account. 
```

Comment:
We don't have statement example so maybe re-phrase this?

Reply:
@ZoranPandovski I moved this below the example `CREATE DATABASE` statement.